src/widget/form/settings/avform.cpp

```diff
@@ -116,6 +117,12 @@ void AVForm::showEvent(QShowEvent* event)
     createVideoSurface();
     getVideoDevices();
 
+    // To don't block GUI thread.
+    QtConcurrent::run([this]()
+    {
+        camVideoSurface->setSource(&camera);
```

Comment:
@sudden6 Yes, but the fact that the old behavior comes back... is weird, so, I thought of something weird ^^